src/types/fieldExtractor.ts

```diff
@@ -9,6 +10,7 @@ export type ExtractorTypes = {
 
 export enum TitleExtractorTypes {
   METADATA_NAME = 'metadata.name',
+  METADATA_NAME_WITHOUT_LEADING_INFO = 'metadataNameWithoutLeadingInfo',
```

Comment:
I also considered `metadataNameWithoutTrackNumbers` but this seems slightly more explicit